pi_local.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This script will log in the address of a raspberry-pi 400 ubuntu server, copy the files over the Pi, excute the script, and SSH into the server.
+PI_HOST=pi.attlocal.net
```

Comment:
Figure out how to access an env var in the code but also provide a default if the env var is not provided or set. Then you can let the user declare the PI_HOST outside of the script like this:

`PI_HOST=mypi.dejavustockton.com bash pi_local.sh`

If they don't provide an FQDN then it will use the default.

Look up default values when accessing an env var bash.